play-json/shared/src/main/scala/play/api/libs/json/JavaEnumFormat.scala

```diff
@@ -0,0 +1,34 @@
+package play.api.libs.json
+
+/**
+ * Helper for format a java enum into or from json
+ *
+ * public enum TestEnum {
+ *
+ *   TEST1,
+ *   TEST2,
+ *   OKAY
+ *
+ * }
+ *
+ * implicit val  TestEnumFormat = JavaEnumFormat.format[TestEnum]
+ *
+ * @author Sebastian Hardt (s.hardt@micromata.de)
+ */
+object JavaEnumFormat {
```

Comment:
With not with other default? (Take care about `@author` vs CLA)

Reply:
Sorry about this will remove it :)